rename.cc

```diff
@@ -0,0 +1,54 @@
+#include <fcntl.h>
+#include <sys/stat.h>
+
+#include "./capsicum-test.h"
+#define RENAME_TEST_FILENAME        ".rename_test"
+#define RENAMEAT_TEST_FILENAME      ".renameat_test"
+#define RENAMEAT_TEST_DIR           ".renameat_testdir"
```

Comment:
Is there a need for these to be .hidden?  The other tests generally create visible files; it makes it easier to spot when a test hasn't cleared up after itself.

Also, the existing tests tend to just use strings inline rather than having `#define`s for them, so probably more consistent to do the same.